HerPublicWebsite/Models/Questionnaire/SelectAddressViewModel.cs

```diff
@@ -0,0 +1,11 @@
+using System.Collections.Generic;
+using GovUkDesignSystem.Attributes.ValidationAttributes;
+
+namespace HerPublicWebsite.Models.Questionnaire;
+
+public record class SelectAddressViewModel : QuestionFlowViewModel
+{
+    public List<OsPlacesResult> Addresses {get; set;}
+    [GovUkValidateRequired(ErrorMessageIfMissing = "You must select an address")]
+    public string Index {get; set;}
```

Comment:
I think this can be an `int?` instead of a `string` so you don't have to manually convert it later on?

Reply:
Also can we call it something a bit more descriptive? e.g. SelectedAddressIndex

Reply:
Making it an `int?` doesn't work because the `GovUkRadiosFromStringsFor` method requires it to be a string. It probably wouldn't be difficult to make a `GovUkRadiosFromSerializableFor` which would work for all serializable data?